style.scss

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: 'Archivo Black', sans-serif;
+  font-size: 10px;
+  background-image: url("stars-background.jpg");
+  color: white;
+}
+
+h1 {
+  display: none;
+}
+
+h2 {
+  font-size: 3rem;
+  margin: 0;
+}
+
+h3 {
+  font-size: 2rem;
+  margin: 0;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 50px;
+  padding: 20px;
+
+  #title-image {
+    padding-top: 50px;
+    width: 100%;
+  }
+
+  .sun-image {
+    filter: drop-shadow(0 0 50px #ef5f02);
+    width: 100%;
+    animation: glow-animation 2s linear infinite;
+  }
+
+  @keyframes glow-animation {
+    0% {
+      filter: drop-shadow(0 0 50px #ef5f02);
+    }
+    50% {
+      filter: drop-shadow(0 0 50px #c31b02);
+    }
+    100% {
+      filter: drop-shadow(0 0 50px #ef5f02);
+    }
+  }
+}
+
+.planet-card {
+  width: 90%;
```

Comment:
should be padding on the sides instead of width

Reply:
done